main/src/main/scala/sbt/internal/server/NetworkChannel.scala

```diff
@@ -176,7 +179,11 @@ final class NetworkChannel(
           try {
             val onHeader: String => Unit = line => {
               if (line.startsWith(ct) && line.contains(x1)) {
-                logMessage("error", s"server protocol $x1 is no longer supported")
+                logMessage(
+                  "error",
+                  s"server protocol $x1 is no longer supported",
+                  Some("run:error")
```

Comment:
Can we remove new line?

Reply:
Can't remove the newline, since it's formatter-dictated